.github/workflows/daily-link-checker.yml

```diff
@@ -0,0 +1,31 @@
+name: Check all Markdown links daily
```

Comment:
As the main branch is not `v2` in the github repo, this will also require a "dummy" workflow checked in in the main `master` branch in a follow up PR.  check here for an example: https://github.com/SeldonIO/seldon-core/blob/master/.github/workflows/license.yml